lms/envs/production.py

```diff
@@ -1175,14 +933,3 @@ def get_env_setting(setting):
 ########################## Derive Any Derived Settings  #######################
 
 derive_settings(__name__)
-
-######################### Overriding custom enrollment roles ###############
-
-MANUAL_ENROLLMENT_ROLE_CHOICES = ENV_TOKENS.get('MANUAL_ENROLLMENT_ROLE_CHOICES', MANUAL_ENROLLMENT_ROLE_CHOICES)
-
-########################## limiting dashboard courses ######################
-
-DASHBOARD_COURSE_LIMIT = ENV_TOKENS.get('DASHBOARD_COURSE_LIMIT', None)
```

Comment:
Is it different to ensure that this variable exists with a default of None?  I guess we can see how it is used.

Reply:
It's defaulting `None` when it gets looked up too. https://github.com/edx/edx-platform/blob/20e02dd48f7056d683b8ae33e71153120506c569/common/djangoapps/student/views/dashboard.py#L549

Reply:
👍